sdk/src/main/java/com/qonversion/android/sdk/internal/common/localStorage/LocalStorage.kt

```diff
@@ -0,0 +1,21 @@
+package com.qonversion.android.sdk.internal.common.localStorage
+
+interface LocalStorage {
+
+    fun putInt(key: String, value: Int)
+
+    fun getInt(key: String, defValue: Int): Int
+
+    fun putFloat(key: String, value: Float)
+
+    fun getFloat(key: String, defValue: Float): Float
+
+    fun putLong(key: String, value: Long)
+
+    fun getLong(key: String, defValue: Long): Long
+
+    fun putString(key: String, value: String)
+
+    fun getString(key: String, defValue: String): String
```

Comment:
Default values for getters should be optional, I don't need to pass them always. And thus return values should be nullable for the cases when no value is stored and no default value is provided

Reply:
It's not possible because of Android SharedPreferences implementation